src/main/cpp/commands/SpinIT.cpp

```diff
@@ -0,0 +1,21 @@
+#include "commands/SpinIT.h"
+#include "../../include/RobotContainer.h"
+
+SpinIT::SpinIT(IT* it, std::function<double()> rightTrigger) : m_it{it}, rightTrig{rightTrigger} {
+    AddRequirements({it});
+}
+void SpinIT::Initialize() {
+    
+}
+void SpinIT::Execute() {
+    if (abs(rightTrig()) > .05) {
```

Comment:
Use defines for "magic numbers". What I mean by this is that any hard coded value inside your code should be clearly defined at the top of the file. Allows for easy configuration and live changes.

Example:
At the top of the file after the includes, do:
```
#define DEADBAND 0.05
```
Then in your logic:
```
if (abs(rightTrig()) > DEADBAND) {
```